status/status.go

```diff
@@ -0,0 +1,91 @@
+package status
+
+import "sync"
+
+// Status represents a status update from an action within Empire.
+type Status interface {
```

Comment:
Probably not much advantage to making this an interface, so we might as well just go with a struct like:

```go
type Status struct {
        // A friendly human readable message about the status change.
        Message string
}

// String implements the fmt.Stringer interface.
func (s *Status) String() string {
        return s.Message
}
```

It may make sense to eventually add a `Status` field to that struct, which we could use to update a field on the `Release` and show the status in `emp releases` too.